pkg/iac-providers/kustomize/v3/testdata/simple-deployment/deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
```

Comment:
One suggestion--can we keep any tests common for v2/v3/v4+ in a common dir, but leave any version-specific ones in the v2/v3/v4 dirs? This would just help keep maintenance tasks down